conf/app.routes

```diff
@@ -12,7 +12,102 @@ GET        /check-your-answers                          controllers.CheckYourAns
+POST       /changeSimilarItemCommodityCode                  controllers.SimilarItemCommodityCodeController.onSubmit(mode: Mode = CheckMode)
+
+GET        /commodityCodeRulingReference                        controllers.CommodityCodeRulingReferenceController.onPageLoad(mode: Mode = NormalMode)
+POST       /commodityCodeRulingReference                        controllers.CommodityCodeRulingReferenceController.onSubmit(mode: Mode = NormalMode)
+GET        /changeCommodityCodeRulingReference                  controllers.CommodityCodeRulingReferenceController.onPageLoad(mode: Mode = CheckMode)
+POST       /changeCommodityCodeRulingReference                  controllers.CommodityCodeRulingReferenceController.onSubmit(mode: Mode = CheckMode)
+
+GET        /legalChallenge               controllers.LegalChallengeController.onPageLoad(mode: Mode = NormalMode)
+POST       /legalChallenge               controllers.LegalChallengeController.onSubmit(mode: Mode = NormalMode)
+GET        /changeLegalChallenge                  controllers.LegalChallengeController.onPageLoad(mode: Mode = CheckMode)
+POST       /changeLegalChallenge                  controllers.LegalChallengeController.onSubmit(mode: Mode = CheckMode)
+
+GET        /legalChallengeDetails                        controllers.LegalChallengeDetailsController.onPageLoad(mode: Mode = NormalMode)
+POST       /legalChallengeDetails                        controllers.LegalChallengeDetailsController.onSubmit(mode: Mode = NormalMode)
+GET        /changeLegalChallengeDetails                  controllers.LegalChallengeDetailsController.onPageLoad(mode: Mode = CheckMode)
+POST       /changeLegalChallengeDetails                  controllers.LegalChallengeDetailsController.onSubmit(mode: Mode = CheckMode)
+
+GET        /supportingInformation               controllers.SupportingInformationController.onPageLoad(mode: Mode = NormalMode)
+POST       /supportingInformation               controllers.SupportingInformationController.onSubmit(mode: Mode = NormalMode)
+GET        /changeSupportingInformation                  controllers.SupportingInformationController.onPageLoad(mode: Mode = CheckMode)
+POST       /changeSupportingInformation                  controllers.SupportingInformationController.onSubmit(mode: Mode = CheckMode)
+
+GET        /supportingInformationDetails                        controllers.SupportingInformationDetailsController.onPageLoad(mode: Mode = NormalMode)
+POST       /supportingInformationDetails                        controllers.SupportingInformationDetailsController.onSubmit(mode: Mode = NormalMode)
+GET        /changeSupportingInformationDetails                  controllers.SupportingInformationDetailsController.onPageLoad(mode: Mode = CheckMode)
+POST       /changeSupportingInformationDetails                  controllers.SupportingInformationDetailsController.onSubmit(mode: Mode = CheckMode)
```

Comment:
URLs are case-insensitive so these may end up looking like `.../gov.uk/changesupportinginformationdetails`

Reply:
Scaffold generated. Ideally better to display as `change-supporting-information-details`